shaal/DrupalPod
# Running composer with --prefer-source

# The Problem/Issue/Bug
`repos/drupal` structure is complex/limited

## How this PR Solves The Problem
Any Drupal project, will be installed by using that project's git repo (source code)
By adding to `composer.json` this section
```
   "config": {
        "preferred-install": {
            "drupal/*": "source"
        }
    },
```

## Manual Testing Instructions
To test:
1. Open this PR (using the button "open with Gitpod" below)
1. `rm composer.lock` (because prebuild already installed Drupal ^9.2)
1. `composer require drupal/core:^8.9`
(You can change `^8.9` with any version you want to install)

1. `cd web/core`
1. Type `git remote -v`
1. Confirm it is using Git source files.

## Related Issue Link(s)

## Release/Deployment notes



<a href="https://gitpod.io/#https://github.com/shaal/DrupalPod/pull/25"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
